docs/ai/conceptual/prompt-engineering-in-dot-net.md

```diff
@@ -0,0 +1,190 @@
+First president to be elected to a second term in office.
+First president to receive votes from every presidential elector in an election.
+First president to fill the entire body of the United States federal judges; including the Supreme Court.
+First president to be declared an honorary citizen of a foreign country, and an honorary citizen of France.
+John Adams ...'" //Text truncated;
+```
+
+### Supporting content
+
+Supporting content is text that an instruction uses as input but isn't the subject of the instruction. The instruction must refer to the supporting content. As with primary content, supporting content should be clearly distinct from the instruction it supports.
+
+Suppose you use the instruction **"Summarize US Presidential accomplishments"** to produce a list. The model might organized and order it in any number of ways. But what if you want the list to group the accomplishments by a specific set of categories? You could adjust your instruction by appending **"&nbsp;grouped by category"** to it, but a model is unlikely to correctly determine which specific categories you want.
+
+To make sure the model uses the categories you want, you could append supporting content to specify your categories and adjust your instruction accordingly. You could add a line of supporting content below the instruction and then change the instruction so it refers to the supporting content:
+
+```csharp
+prompt = @$"Instructions: Summarize US Presidential accomplishments, grouped by category.
+Categories: Domestic Policy, US Economy, Foreign Affairs, Space Exploration.
+Accomplishments: George Washington
+First president of the United States.
+First president to have been a military veteran.
+First president to be elected to a second term in office.
+First president to receive votes from every presidential elector in an election.
+First president to fill the entire body of the United States federal judges; including the Supreme Court.
+First president to be declared an honorary citizen of a foreign country, and an honorary citizen of France.
+John Adams ..."; //Text truncated
+```
+
+## Examples
+
+This section explains the use of examples in .NET prompt engineering.
+
+An example is text that shows the model how to respond by providing sample user input and model output. The model uses examples to infer what to include in completions. Examples can come either before or after the instructions in an engineered prompt, but the two shouldn't be interspersed.
+
+Like a normal GPT interaction, an example starts with a prompt. The example can include a completion but it's not required. A completion in an example doesn't have to include the verbatim response&mdash;it might just contain a formatted word, the first bullet in an unordered list, or something similar to indicate how each completion should start.  
+
+Examples are classified as zero-shot learning or few-shot learning based on whether they contain verbatim completions.
```

Comment:
```suggestion
Examples are classified as [zero-shot learning or few-shot learning](zero-shot-learning.md) based on whether they contain verbatim completions.
```
Figure it might be good to link to the docs with more details. But, I could see putting this link somewhere else nearby.

Reply:
:)